config/index.js

```diff
@@ -472,6 +472,51 @@ var conf = convict({
       default: true,
       env: 'SECURITY_HISTORY_ENABLED'
     }
+  },
+  signinUnblock: {
+    codeLength: {
+      doc: 'Number of base36 digits to make up an unblockCode',
+      default: 8,
+      env: 'SIGNIN_UNBLOCK_CODE_LENGTH'
+    },
+    codeLifetime: {
+      doc: 'How long an unblockCode should be valid for',
+      format: 'duration',
+      default: '1 hour',
+      env: 'SIGNIN_UNBLOCK_CODE_LIFETIME'
+    },
+    enabled: {
+      default: true
+    },
+    allowEmailRegex: {
+      doc: 'If feature enabled, allow sign-in unblock for email addresses matching this regex.',
+      format: Array,
+      default: [
+        '.+@mozilla\.com$'
+      ],
+      env: 'SIGNIN_UNBLOCK_ALLOW_EMAIL_REGEX'
+    },
+    sample_rate: {
```

Comment:
Snake case definitely sticks out. Only other reference I found was in content-server [configs](https://github.com/mozilla/fxa-content-server/blob/master/server/lib/configuration.js#L405). More generally, seems like config casing is different among services and use both types :(